sklearn/tests/test_docstring_parameters.py

```diff
@@ -299,6 +299,10 @@ def test_fit_docstring_attributes(name, Estimator):
     else:
         est.fit(X, y)
 
+    if Estimator.__name__ == "BisectKMeans":
+        skipped_attributes.add("n_features_in_")
```

Comment:
Running checks failing on that, because there's no `n_features_in` in `BisectKMeans` docstring (I think there's not even in `KMeans`).
I've seen no point in adding attribute that is not used there. If you think It would be better that way - I may change that.

Reply:
I see. This attribute was added recently in #20228 for KMeans (see more info in [SLEP10](https://scikit-learn-enhancement-proposals.readthedocs.io/en/latest/slep010/proposal.html)). We should add it as well.

Reply:
Resolved with next commit